components/search_results_item/search_results_item.jsx

```diff
@@ -287,6 +287,7 @@ export default class SearchResultsItem extends React.PureComponent {
                         searchStyle={'search-item__comment'}
                     />
                     <a
+                        href='#'
```

Comment:
You'll need to add an `e.preventDefault` to `this.handleJumpClick` so that it doesn't add a random `#` to the address bar when this is clicked on